Jenkinsfile

```diff
@@ -38,7 +38,7 @@ node('ios-osx') {
         sh '''#!/bin/bash -l
             set -x
             set -e
-            rm -R ./node_modules
+            rm -Rf ./node_modules
```

Comment:
while using `yarn` deleting node_modules should not be necessary. Can we please try to remove it and see how it goes?